.devcontainer/devcontainer.json

```diff
@@ -20,21 +20,16 @@
   "customizations": {
     "vscode": {
       "extensions": [
+        "GitHub.codespaces",
+        "charliermarsh.ruff",
         "ms-python.python",
-        "ms-python.isort",
-        "ms-python.black-formatter",
-        "ms-python.vscode-pylance",
```

Comment:
Got rid of pylance since we aren't currently using it anywhere else. Also, haven't tested this file since I don't know how to use it.